kitsune/motidings/tests/__init__.py

```diff
@@ -0,0 +1,10 @@
+from factory.django import DjangoModelFactory
+from tidings.models import Watch
+
+
+class WatchFactory(DjangoModelFactory):
+    class Meta:
+        model = Watch
+
+    event_type = 'fooevent'
+    is_active = True
```

Comment:
I think I'd make secret a [fuzzy text value](http://factoryboy.readthedocs.org/en/latest/fuzzy.html#fuzzytext), so it is different each time. I think the case where we don't want a secret on a Watch is rarer than the case where we want a secret.